ccmlib/node.py

```diff
@@ -1277,7 +1281,10 @@ def stress_process(self, stress_options, **kwargs):
                 # specify used jmx port if not already set
                 if not [opt for opt in stress_options if opt.startswith('jmx=')]:
                     stress_options.extend(['-port', 'jmx=' + self.jmx_port])
-
+        if self.tablets_enabled():
+            stress_options = self._set_keyspace_initial_tablets(
+                stress_options, datacenter=self.data_center, rf=min(len(self.cluster.nodes), 3)
```

Comment:
maybe be the default rf should be calculated this way in `_set_keyspace_initial_tablets`
so the logic could be reused in the future if and when called from other places

Reply:
yes, at first I wanted also this way, but to ease unit testing i made it this way. I moved this calculation to `cluster` object. WDYT?